NOLF2/ClientShellDLL/TO2/ScreenDisplay.cpp

```diff
@@ -688,4 +705,30 @@ void CScreenDisplay::ConfirmHardwareCursor(LTBOOL bReturn)
 	UpdateData(LTFALSE);
 }
 
+void CScreenDisplay::UpdateHelpText()
```

Comment:
minor issue: isn't this a bit terrible...and it should be added as a translatable resource?

Reply:
Addressed in https://github.com/Katana-Steel/lithtech/pull/55/commits/ce97a89a074ba8c065b5515103eed6d29a2814b5